.github/workflows/tests.yaml

```diff
@@ -70,6 +70,7 @@ jobs:
       run: |
         pip install --upgrade pip wheel
         pip install -e ".[dev]"
+        pip install "git+https://github.com/efiop/dvc-objects.git@db-init-lock"
```

Comment:
@efiop the main error is happening here, the link you wanna run can't be found, please check if `https://github.com/efiop/dvc-objects.git@db-init-lock` is private

Reply:
@ArshErgon Not sure what you are looking at, but that's not true for test runs. It does install it, but tests fail.

Reply:
@ArshErgon it's a pip git URL, not an HTTP address. If you try to go directly to `https://github.com/efiop/dvc-objects.git@db-init-lock` in a web browser it will 404 because that does not exist, but that has nothing to do with this PR. pip does not handle it as an HTTP address, pip installs from a git repo URL + branch/tag name.